src/components/events/AdminEvents/AdminEvents.jsx

```diff
@@ -64,32 +64,41 @@ const AdminEvents = () => {
   const determineStatus = startDatetime =>
     new Date(startDatetime) > new Date() ? 'upcoming' : 'past';
 
+  const getEventsByStatus = (events, status) => {
+    let filteredEvents = events;
+    if (status === 'all') {
```

Comment:
this is passable, but for future reference, a more concise way of doing this is:
```
const filteredEvents = {
"all": events,
"upcoming": events.filter(event => determineStatus(event.startDatetime) === 'upcoming',
"past": ...
};
return filteredEvents[status];
```